packages/drawer/src/mwc-drawer.ts

```diff
@@ -86,6 +94,7 @@ export class Drawer extends BaseElement {
       },
       trapFocus: () => {
         document.$blockingElements.push(this);
+        this.appContent.inert = true;
```

Comment:
What about just making `aside.mdc-drawer` the blocking element? e.g.
```
document.$blockingElements.push(this.mdcRoot);
```
Does that work?

Reply:
The scrim is a sibling of the aside, and has to be clickable unfortunately.

Reply:
(And mdc guidelines require the `.mdc-drawer-app-content` be a sibling of the `.mdc-drawer` and `.mdc-drawer-scrim`)

Reply:
I see. Thanks for the explanation.